ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -16904,7 +16904,7 @@ ABoundActivity * HqlCppTranslator::doBuildActivityCreateRow(BuildCtx & ctx, IHql
     IHqlExpression * self = selfCursor->querySelector();
 
     if (isDataset)
-        associateSkipReturnMarker(funcctx, queryBoolExpr(false), selfCursor);
+        associateSkipReturnMarker(funcctx, queryZero(), selfCursor);
```

Comment:
The transform function that it is generating - which returns 0 (not false) if it is to skip.